frog/imports/ui/GraphEditor/store/operator.js

```diff
@@ -90,10 +89,22 @@ export default class Operator {
     };
   }
 
-  @computed get dragPoint(): [number, number] {
+  @computed get dragPointTo(): [number, number] {
+    return [this.x + 30, this.y + 30];
+  }
+
+  @computed get dragPointFrom(): [number, number] {
+    return this.dragPointTo;
```

Comment:
`dragPointFrom` is the same as `dragPointTo`, so we don't distinguish the input and output of operators. I would suggest the following: 
```Javascript
@computed get dragPointTo(): [number, number] {
    return [this.x, this.y + 30];
...
@computed get dragPointTo(): [number, number] {
    return [this.x + 60, this.y + 30];
```

No need to implement it, just tell me if you agree with it so I'll do something like this when working on drawing the paths.